manifests/s/Supernova/SupernovaPlayer/0.92/Supernova.SupernovaPlayer.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.3.0
```

Comment:
```suggestion
# Created with komac v2.3.0 - new commit for testing
```
@Dragon1573 - the Policy Service team is asking for a new commit to be added so they can track down the cause of #142854; Can you commit this suggestion so they can continue their testing?